### PR TITLE
EDSC-842 Keep browse image ratio in collection and granule list views.

### PR DESCRIPTION
**Pull request by mightynimble**

### CODE REVIEW

File: app/assets/stylesheets/modules/panel_list.css.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-$panel-list-item-height: 85px;
+$panel-list-item-height: 75px;
```
Comment (bilts) [inline app/assets/stylesheets/modules/panel_list.css.scss]: We're requesting a 65px image from the browse scaler, but it ends up being drawn at 75px on the screen. Please update the browse scaler URL to request the correct size.

File: app/assets/stylesheets/modules/panel_list.css.scss [hunk 2]
```diff
@@ -80,12 +80,18 @@ $panel-list-thumbnail-size: $panel-list-item-height;
...
   position:relative;
   margin-right: .5em;
+  width: $panel-list-thumbnail-size + 10px;
+  height: $panel-list-thumbnail-size + 10px;
+  background-color: #222;
+  border: 5px solid #323232;
 }
 
 .panel-list-thumbnail {
   height: $panel-list-thumbnail-size;
```
Comment (bilts) [inline app/assets/stylesheets/modules/panel_list.css.scss]: What happens when images are wider than they are tall?

Comment (mightynimble) [inline app/assets/stylesheets/modules/panel_list.css.scss]: The ```max-width``` below will handle that.

File: app/assets/stylesheets/modules/panel_list.css.scss (hunk 1 above)
Comment (mightynimble) [inline app/assets/stylesheets/modules/panel_list.css.scss]: I see we are requesting *85px* images in ```collection.coffee```:
```
      if collection_id?
        "#{scalerUrl}/datasets/#{collection_id}?h=85&w=85"
      else if granule?
        "#{scalerUrl}/granules/#{granule}?h=85&w=85"
```
Should I update this to 75px? Or you are referring somewhere else?

Comment (bilts) [inline app/assets/stylesheets/modules/panel_list.css.scss]: That's on the collection. I was looking at the granule thumbnail. Both of them probably need to be fixed.

File: app/assets/stylesheets/modules/panel_list.css.scss (hunk 2 above)
Comment (bilts) [inline app/assets/stylesheets/modules/panel_list.css.scss]: Change line 17 of granule.coffee from:

    "#{scalerUrl}/granules/#{@id}?h=#{h}&w=#{w}"

to:

    "http://placehold.it/30x65"

Which is an image taller than it is wide. The code looks correct.  Now change it to:

    "http://placehold.it/65x30"

Which is an image wider than it is tall. The code stretches it into a square. It should be centered and maintain aspect ratio.

**Comment by bilts** [comment]
The "No Image Available" image on collections is broken with this change

File: app/assets/javascripts/models/data/collection.js.coffee [hunk 3]
```diff
@@ -125,9 +125,9 @@ ns.Collection = do (ko
       granule = @browseable_granule
       collection_id = @id for link in @links when link['rel'].indexOf('browse#') > -1
       if collection_id?
-        "#{scalerUrl}/datasets/#{collection_id}?h=85&w=85"
+        "#{scalerUrl}/datasets/#{collection_id}?h=75&w=75"
```
Comment (bilts) [inline app/assets/javascripts/models/data/collection.js.coffee]: Can you define the dimension in one place and use that constant? I'm uncomfortable of sprinkling the magic number 75 everywhere. Also, it seems like we've failed to update it consistently in the past.

Comment (mightynimble) [inline app/assets/javascripts/models/data/collection.js.coffee]: I will put it in config.js.coffee.erb.

**Comment by bilts** [comment]
This still breaks the layout of "No Image Available" albeit less so (compare to ops, which has no light gray border). Also, the GIBS and NRT badges are no longer appearing.

**Comment by bilts** [comment]
GIBS and NRT badges appear below thumbnails in the collections list

**Comment by bilts** [comment]
Landscape-oriented images are not quite vertically centered, probably due to extra whitespace (Please solve for real, don't just subtract 3 from the line-height).

**Comment by mightynimble** [comment]
Fixed "No Image Available" borders
Fixed GIBS and NRT badges.
About the vertical-align issue: it seems that it works as it should be. Take ASTER L1A Reconstructed Unprocessed Instrument Data V003 as an example here:
The browse is 70px tall which left 5px blank space. Vertical-align divides the 5 into 3 and 2 px. If you zoom in the page so that you can measure it, you will see that the top is 1px (2px if you zoom in 2X) taller than the bottom.

**Pull request merged**